src/ratbag_emu/protocol/util/profile.py

```diff
@@ -15,13 +15,19 @@ class Profile(object):
     ]
     step = 100
 
-    report_rate = 1000  # Hz
+    active_report_rate = 0
+    report_rates = [ # Hz
+        1000
+    ]
 
+    # TODO: Add proper LED structure
```

Comment:
for simple structs use [`namedtuple`](https://docs.python.org/3/library/collections.html)

```
Led = namedtuple('Led', ['red', 'green', 'blue'])
l1 = Led(0xff, 0xff, 0xff)
```
Useful if you think you'll eventually implement a class for these anyway, you won't have to change any of the caller code.

Reply:
Sure.